### PR TITLE
Update kyc-match.yaml to include API description in yaml

### PR DESCRIPTION
**Pull request by ToshiWakayama-KDDI**
To include API description in YAML

#### What type of PR is this?

* correction
* cleanup
* documentation

#### What this PR does / why we need it:

* To align with CAMARA common rules

#### Which issue(s) this PR fixes:

* Issue #37

### CODE REVIEW

**Review by GillesInnov35 — APPROVED** [review]
Thanks

File: code/API_definitions/kyc-match.yaml [hunk 1]
```diff
@@ -2,6 +2,39 @@ openapi: 3.0.3
...
+
+    * Before calling this API, 3rd parties / enterprise customers who want to use this API should make contact with API provider/ MNO for use of this API.  As that will depend on each API provider / MNO's business process as well as GSMA Open Gateway standard process, it is out of scope of this API definition.
+
+    * When calling this API, at the beginning, there should be required processes for Authentication / Authorisation / End User Consent capturing.  As those processes are defined as CAMARA commonality standards, they are out of scope of this API definition, however, use of the OpenID Connect (OIDC) is stated as security scheme.
+
+    ## Resources and Operations overview
+
+    The API provides the following endpoint:
+
+    * An endpoint to verify the matching of a number of attributes related to a mobile phone user identity against the acount data bound to their phone number.
```
Comment (fernandopradocabrillo) [inline code/API_definitions/kyc-match.yaml:35]:
```suggestion
    * An endpoint to verify the matching of a number of attributes related to a mobile phone user identity against the acount data bound to their mobile phone number.

```

**Review by fernandopradocabrillo — APPROVED** [review]
LGTM thanks @ToshiWakayama-KDDI !

**Comment by ToshiWakayama-KDDI** [comment]
Merged as agreed at 2024/02/06 meeting.

**Pull request merged**